### PR TITLE
New icons

### PR DESCRIPTION
**Pull request by goranalkovic-infinum**
Adds new icons for Eigthshift forms to be used in the future.

image

### CODE REVIEW

**Review by dingo-d — DISMISSED** [review]
I've noticed that the 'prefix' is `eightshift`, should we maybe shorten this to `es`?

**Review by gabriel-glo — DISMISSED** [review]

**Comment by goranalkovic-infinum** [comment]
> I've noticed that the 'prefix' is `eightshift`, should we maybe shorten this to `es`?

I can change it, not a problem!

Is `eightshift-` -> `es-` and `eightshift-forms-` to `esf-` ok?

**Review by kmaric69 — APPROVED** [review]

**Review by dingo-d — APPROVED** [review]

**Review by gabriel-glo — APPROVED** [review]

**Review by IvanGrginovInf — APPROVED** [review]
